apps/experiments/migrations/0050_bootstrap_voice_providers.py

```diff
@@ -40,5 +40,5 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        migrations.RunPython(bootstrap_voice_providers)
+        migrations.RunPython(bootstrap_voice_providers, elidable=True)
```

Comment:
Hmm, should this be marked as `elidable`?

Reply:
ah, no. Thanks

Reply:
Actually this one is fine since it's a one-off migration that's not required on new installs.

Reply:
My bad, you're right! I think I confused this with the data migrations that loaded the voices.